blueprints/dotnet/src/APIGatewayAuthorizerHandler.Tests/IntegrationTests.cs

```diff
@@ -0,0 +1,37 @@
+using System.Linq;
+using Amazon.Lambda.TestUtilities;
+using APIGatewayAuthorizerHandler.Model;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace APIGatewayAuthorizerHandler.Tests
+{
+    public class IntegrationTests
+    {
+        [Fact]
```

Comment:
This is just a really basic integration test to help attach the debugger, I'd like to add proper unit testing later, but for now this allows for a smoke test and lets the developer manually test at least a little bit.